src/reconstruction/refinement/addReaction.m

```diff
@@ -1,4 +1,4 @@
-function [model, rxnIDexists] = addReaction(model, rxnName, metaboliteList, stoichCoeffList, revFlag, lowerBound, upperBound, objCoeff, subSystem, grRule, geneNameList, systNameList, checkDuplicate, printLevel)
+function [model, rxnIDexists] = addReaction(model, rxnID, varargin) %function [model, rxnIDexists] = addReaction(model, rxnName, metaboliteList, stoichCoeffList, revFlag, lowerBound, upperBound, objCoeff, subSystem, grRule, geneNameList, systNameList, checkDuplicate, printLevel)

```

Comment:
Do we still need the old header here? Or could that be moved to the bottom of the function header?